code/pizzabot.js

```diff
@@ -4,4 +4,73 @@ const pepperoni = "Pepperoni Pizza"
 
 const pizzaPrice = 80
 
-//Put your Javscript code here:
+
+// validateOrderName() which should take the orderName variable as an argument and return true or false if the pizza is on the menu or not.
+// const validateOrderName = (orderName) => {
+//   if ((orderName === hawaiian) || (orderName === vegetarian) || (orderName === pepperoni)) {
+//     return true;
+//   } else {
+//     return false;
+//   }
+// }
+
+// calculateTotalCost() which takes orderQuantity and pizzaPrice as arguments and returns the total cost for the order.
+const calculationTotalCost = (orderQuantity, pizzaPrice) => {
+  return orderQuantity * pizzaPrice;
+}
+
+// calculateCookingTime() which takes orderQuantity and returns the number of minutes it will take to finish the order.
+const calculateCookingTime = (orderQuantity) => {
+  if (orderQuantity <= 2) {
+    return 10;
+  } else if (orderQuantity > 2 && orderQuantity <= 5) {
+    return 15;
+  } else {
+    return 20;
```

Comment:
Great job :)
Regarding clean code, there is also another convention, let me introduce you to it.
Apart from returning given thing from function, `return` keyword does one more thing - it automatically finishes function. That's why we mustn't write anything below `return`, because that code will never be executed (of course in your case we have code below return true, but it's inside if statement, so it wouldn't enter second statement in the first place).
Thanks to that feature of `return` keyword, we can write our function in slightly different, far more clean fashion :
```
const calculateCookingTime = (orderQuantity) => {
  if (orderQuantity <= 2) {
    return 10;
  } else if (orderQuantity > 2 && orderQuantity <= 5) {
    return 15;
  }
  return 20;
}
```
As you can see, we are not writing `else` statement anymore. We can do that thanks to behaviour mentioned above; if `orderQuantity` is lower than 6, we will execute return true code from line 25 or 27 and finish function. If, on the other hand, `orderQuantity` is higher or equal 6, we will never enter line 25 or 27 and jump straight into line 29. After executing line 29 we will finish function same way :)